internal/bits/bit_array.go

```diff
@@ -193,6 +216,15 @@ func (bA *BitArray) not() *BitArray {
 	for i := 0; i < len(c.Elems); i++ {
 		c.Elems[i] = ^c.Elems[i]
 	}
+
+	// Flip count is simply total bits minus current true bits
+	c.TrueBitCount = c.Bits - c.TrueBitCount
+
+	// Clear any excess bits in the last element if necessary
```

Comment:
Where's this coming from?

Reply:
Sorry, I meant that we were not clearing any excess bits previously.

Reply:
Yeah maybe it isn't really necessary, I have removed